src/main/controllers/helpers/ApplicationStateHelper.ts

```diff
@@ -0,0 +1,30 @@
+import { YesOrNo } from '../../definitions/case';
+import { GenericTseApplicationTypeItem } from '../../definitions/complexTypes/genericTseApplicationTypeItem';
+import { HubLinkStatus } from '../../definitions/hub';
+
+export const getNewApplicationStatus = (selectedApplication: GenericTseApplicationTypeItem): HubLinkStatus => {
+  const currentStatus = selectedApplication.value.applicationState;
```

Comment:
inline into the switch (maybe? up to you what's nicer to read)

Reply:
Liked the currentStatus variable name as it made the switch easier to read but agree it is a useless assignment in a sense. I've made the change but added a description of what the switch statement as a whole is doing